modules/machineset-yaml-osp.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+//
+// * machine_management/creating-infrastructure-machinesets.adoc
+// * machine_management/creating_machinesets/creating-machineset-osp.adoc
+
+[id="machineset-yaml-osp_{context}"]
+=  Sample YAML for a MachineSet Custom Resource on {rh-openstack}
+
+This sample YAML defines a MachineSet that runs on
+{rh-openstack-first} and creates nodes that are labeled with
+`node-role.openshift.io/NODE_ROLE: ""`
+
+In this sample, `INFRASTRUCTURE_ID` is the infrastructure ID label that is
+based on the cluster ID that you set when you provisioned
+the cluster, and `NODE_ROLE` is the node label to add.
+
+[source,yaml]
+----
+apiVersion: machine.openshift.io/v1beta1
+kind: MachineSet
+metadata:
+  labels:
+    machine.openshift.io/cluster-api-cluster: INFRASTRUCTURE_ID <1>
+    machine.openshift.io/cluster-api-machine-role: NODE_ROLE <2>
+    machine.openshift.io/cluster-api-machine-type: NODE_ROLE <2>
+  name: INFRASTRUCTURE_ID-NODE_ROLE-REGION <3>
+  namespace: openshift-machine-api
+spec:
+  replicas: NUMBER_OF_REPLICAS
+  selector:
+    matchLabels:
+      machine.openshift.io/cluster-api-cluster: INFRASTRUCTURE_ID <1>
+      machine.openshift.io/cluster-api-machineset: INFRASTRUCTURE_ID-NODE_ROLE-REGION <3>
+  template:
+    metadata:
+      labels:
+        machine.openshift.io/cluster-api-cluster: INFRASTRUCTURE_ID <1>
+        machine.openshift.io/cluster-api-machine-role: NODE_ROLE <2>
+        machine.openshift.io/cluster-api-machine-type: NODE_ROLE <2>
+        machine.openshift.io/cluster-api-machineset: INFRASTRUCTURE_ID-NODE_ROLE-REGION <3>
+    spec:
+      providerSpec:
+        value:
+          apiVersion: openstackproviderconfig.openshift.io/v1alpha1
+          cloudsSecret:
+            name: openstack-cloud-credentials
+            namespace: openshift-machine-api
+          flavor: NOVA_FLAVOR
+          image: GLANCE_IMAGE_NAME_OR_LOCATION
+          internalLoadBalancer: ""
+          kind: OpenstackProviderSpec
+          location: centralus
```

Comment:
where is this coming from?

Reply:
Unclear. Probably from reusing parts of existing MachineSet docs samples. Removing "internalLoadBalancer" and "location."

Reply:
This now matches the example in the BZ ticket, as far as I can tell.